google/auth/_helpers.py

```diff
@@ -21,6 +21,20 @@
 from six.moves import urllib
 
 
+def copy_docstring(source_class):
+    """Decorator that copies the a method's docstring from another class."""
+    def decorator(method):
+        """Decorator implementation."""
```

Comment:
No args or returns section.

Also, I recommend using a `__call__`-able class to decorate rather than a closure (to avoid dynamically created functions). Not required that you do this.